INSTALL.md

```diff
@@ -115,11 +115,14 @@ Install Qiita (this occurs through setuptools' `setup.py` file in the qiita dire
 pip install -e .
 ```
 
-You will also need to install several plugins for Qiita to be fully functional. As they are already included in the repository, to install, simply execute:
+You will also need to install several plugins for Qiita to be fully functional.
+To install those plugins, simply execute
+
 ```bash
-pushd qiita_plugins/target_gene
-pip install -e .
-popd
+pip install https://github.com/qiita-spots/qiita_client
```

Comment:
I would prefer not to add them in the setup.py. The reason why I'm installing them here is because ATM 2 of these plugins depend on Qiita, so it makes sense to have them in the same environment. However, once those plugins don't depend on Qiita, I would like to update the installation instructions so they are not installed in the same environment.